pkg/kubelet/kubelet_getters_test.go

```diff
@@ -63,7 +77,29 @@ func TestKubeletDirs(t *testing.T) {
 	exp = filepath.Join(root, "pods/abc123/plugins/foobar")
 	assert.Equal(t, exp, got)
 
+	got = kubelet.getVolumeDevicePluginsDir()
+	exp = filepath.Join(root, "plugins")
+	assert.Equal(t, exp, got)
+
+	got = kubelet.getVolumeDevicePluginDir("foobar")
+	exp = filepath.Join(root, "plugins", "foobar", "volumeDevices")
+	assert.Equal(t, exp, got)
+
 	got = kubelet.getPodContainerDir("abc123", "def456")
 	exp = filepath.Join(root, "pods/abc123/containers/def456")
 	assert.Equal(t, exp, got)
+
+	got = kubelet.getPodResourcesDir()
+	exp = filepath.Join(root, "pod-resources")
+	assert.Equal(t, exp, got)
+
+	// GetNodeConfig for containerManagerStub returns an empty node config
```

Comment:
I don't think this test and the next one actually test anything. I would remove it or implement a proper stub that can return something meaningful to compare

Reply:
> implement a proper stub that can return something meaningful to compare.

Thanks, sure. I will try it.

Reply:
do you still want to implement it or simply remove tests that validate stubs?

Reply:
I will try them on another PR. So deleted the tests in https://github.com/kubernetes/kubernetes/pull/93723/commits/7d58ead809e97b23bb8f9fba3cefb5ab8cfa81e4.